components/rules/rules.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <link
+      rel="stylesheet"
+      href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+      integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p"
+      crossorigin="anonymous"
+    />
+
+    <link rel="stylesheet" href="https://rapidui.netlify.app/styles/main.css" />
+
+    <link rel="stylesheet" href="./rules.css" />
+    <link rel="stylesheet" href="../../index.css" />
```

Comment:
you can use relative path over here instead of this absolute imports with ../

Reply:
I have used absolute path for my component library. How can I use relative path over here ?